gis/gccview/README.md

```diff
@@ -94,32 +92,47 @@ In the DAG file, the arguments for each layer are stored in dictionaries called
 
 ## Adding new layers to GCC Puller DAG
 1. Identify the mapserver_n and layer_id for the layer you wish to add. You can find COT transportation layers here: https://insideto-gis.toronto.ca/arcgis/rest/services/cot_geospatial2/FeatureServer, where mapserver_n is 2 and the layer_id is in brackets after the layer name.
-2. Add a new entry to "bigdata_layers" or "ptc_layers" dictionaries in [gcc_layers_pull.py](/dags/gcc_layers_pull.py) depending on the destination database. 
-3. If is_audited = True, you must also add a primary key for the new layer to "pk_dict" in [gcc_puller_functions.py](gcc_puller_functions.py).
+2. Add a new entry to "bigdata_layers" or "ptc_layers" dictionaries in airflow's variable depending on the destination database. 
+3. If is_audited = True, you must also add a primary key for the new layer to "gcc_layers" in the corresponding airflow variable.
+
+## Manually fetch layers
```

Comment:
Can you please add a description of the 3 options: audited, partitioned, ... none? Each have different input requirements and audited/partitioned require target table to exist in database already I think.